src/engraving/libmscore/system.cpp

```diff
@@ -254,53 +254,43 @@ void System::adjustStavesNumber(int nstaves)
 }
 
 //---------------------------------------------------------
-//   layoutSystem
-///   Layout the System
+//   systemNamesWidth
 //---------------------------------------------------------
 
-void System::layoutSystem(const LayoutContext& ctx, qreal xo1, const bool isFirstSystem, bool firstSystemIndent)
+qreal System::systemNamesWidth(bool getTotalWidth)
```

Comment:
not quite sure what does it mean if I have "false" value here? What are the expectations of "non-total-width"?

Reply:
It had a meaning during early development of this PR, though I didn't get delete it once I no longer needed it. Deleting that now. Thanks for catching!